coordinator/crypto/crypto.go

```diff
@@ -44,13 +45,23 @@ func GenerateCert(
 		return nil, nil, fmt.Errorf("generating serial number: %w", err)
 	}
 
+	var additionalDNSNames []string
+	var additionalIPs []net.IP
+	for _, name := range dnsNames {
```

Comment:
```suggestion
	var dnsNames []string
	var additionalIPs []net.IP
	for _, name := range subjAltNames {
```
nit: these names would be more precise